src/api/cloud/project/ai/capabilities/serving/presetImage/cloud-project-serving-capabilities-stream-region.v6.service.js

```diff
@@ -0,0 +1,9 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectAiCapabilitiesServingPresetImageV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/ai/capabilities/serving/presetImage', {
```

Comment:
Name of the file doesn't match the targeted API

Reply:
nice catch